packages/generator/tests/acceptance/test_client_generator.py

```diff
@@ -28,7 +28,32 @@ def test___command_line_args___create_client___render_without_error(
                 "--module-name",
                 module_name,
                 "--class-name",
-                "TestMeasurement",
+                "NonStreamingDataMeasurementClient",
+                "--directory-out",
+                temp_directory,
+            ]
+        )
+
+    assert not exc_info.value.code
+    _assert_equal(
+        golden_path / filename,
+        temp_directory / filename,
+    )
+
+
+def test___command_line_args___create_client_for_all_active_measurement___render_without_error(
+    test_assets_directory: pathlib.Path,
+    tmp_path_factory: pytest.TempPathFactory,
+    measurement_service: MeasurementService,
+) -> None:
+    temp_directory = tmp_path_factory.mktemp("measurement_plugin_client_files")
+    golden_path = test_assets_directory / "example_renders" / "measurement_plugin_client"
+    filename = "non_streaming_data_measurement_client.py"
```

Comment:
Please update this test to create multiple measurement plug-in clients and assert the module names, class names and number of files generated.

Reply:
Sure, Updated with creating multiple measurement plug-in clients and asserted the module names and number of files generated. Class names are not returned and so file read check is done in other test.